client/ayon_houdini/plugins/create/create_node_preset.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+"""Creator plugin for creating houdini node presets."""
+from ayon_houdini.api import plugin
+import hou
+
+
+def _update_node_parmtemplate(node, defaults):
```

Comment:
A question: Does this function imprint the data into the instance_node already? (Or it is like the temp data publisher and it won't imprint the instance data?)
i.e. some variables you can find in almost every creator and they are missing in this creator.
```
            instance = CreatedInstance(
                self.product_type,
                product_name,
                instance_data,
                self)
            self._add_instance_to_context(instance)
            self.imprint(instance_node, instance.data_to_store())

```

Reply:
The method `_update_node_parmtemplate` doesn't create the extra ayon parameters created by imprint.
It actually doesn't create any parameters at all. 
It only captures the values of any existent parameters (including ayon extra parameters) from the source node and reassigns these values to the target node.

---

I have that feeling that the method name should be refactored but I couldn't come up with a better name :/

Reply:
I remember @BigRoy has written a one-time publish node(along with the generic nodes for publishing). Maybe he can give some maps on this.